### ISSUE TITLE
dbName value not used when auto seeding database

### ISSUE THREAD

**Issue by lriggle-strib** [bealearts/serverless-mongodb-local#9]
I fully admit I may be missing something when it comes to setting this up. I'm using this plugin in conjunction with the `serverless-offline` plugin, and when I turned everything on for the first time, the database that was created was called `test`, even though I specified in the YAML file that the database be called `interview`.

Below is the contents of my YAML file, please let me know what I'm missing about this.

```yaml
service: apollo-lambda
provider:
  name: aws
  runtime: nodejs14.x
integretion:
functions:
  index:
    # this is formatted as <FILENAME>.<HANDLER>
    handler: index.handler
    events:
      - http:
          path: /
          method: post
          cors: true
      - http:
          path: /
          method: get
          cors: true
plugins:
  - serverless-mongodb-local
  - serverless-offline
custom:
  mongodb:
    instance: # MongoMemoryServer() options and defaults https://github.com/nodkz/mongodb-memory-server#available-options-for-mongomemoryserver
      port: 1234
      dbName: interview
      dbPath: ./db
      storageEngine: wiredTiger # Set with `dbPath` to persists the database between instantiations
    seed:
      auto: true
      dataPath: ./test-data
```

**Comment by DavidBeale** [comment]
Thanks for raising this.

You are correct, the seed is ignoring the dbName. Fix incoming.

**Comment by DavidBeale** [comment]
Should be fixed in v1.5.2
